plugins/nf-nomad/src/test/nextflow/nomad/NomadDSLSpec.groovy

```diff
@@ -57,8 +60,13 @@ class NomadDSLSpec  extends Dsl2Spec{
             protected PluginDescriptorFinder createPluginDescriptorFinder() {
                 return new TestPluginDescriptorFinder(){
                     @Override
+                    protected Manifest readManifestFromDirectory(Path pluginPath) {
+                        def manifestPath= getManifestPath(pluginPath)
+                        final input = Files.newInputStream(manifestPath)
+                        return new Manifest(input)
+                    }
                     protected Path getManifestPath(Path pluginPath) {
-                        return pluginPath.resolve('build/resources/main/META-INF/MANIFEST.MF')
+                        return pluginPath.resolve('build/tmp/jar/MANIFEST.MF')
                     }
```

Comment:
@jagedn , I'm just curious about this one - was this change made in upstream Nextflow as well?

Reply:
yes, I think was due to upgrade gradle to 8.3

Reply:
Ah, okay - then it makes sense 👍